test/integration/roles/test_win_package/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+msi_url: https://ansible-ci-files.s3.amazonaws.com/test/integration/roles/test_win_msi/7z922-x64.msi
```

Comment:
hmm, that must have changed recently (probably just to reduce likelihood of network failures, as I think it used to come from sourceforge, or wherever the 7 zip project is).  my guess would be @mattclay ?